test/javascript/tests/view_errors.js

```diff
@@ -154,7 +154,7 @@ couchTests.view_errors = function(debug) {
           db.view("infinite/infinite_loop");
           T(0 == 1);
       } catch(e) {
-          T(e.error == "os_process_error");
+          T(e.error == "os_process_error" || e.error == "unnamed_error");
```

Comment:
Is this a difference between 1.8.5 and 60? If so, is there any better error message we can emit? `unnamed_error` is a bit worrying, but if we really don't know, there's not much we can do.

Reply:
Yes, they are different. For 1.8.5, we have `os_process_error` while it is `unnamed_error` in 6.0. Took some time to get better error message, but don't get bottom yet. So stayed with this for now.